src/view/translation.js

```diff
@@ -964,4 +964,69 @@ Candy.View.Translation = {
 
         'antiSpamMessage': 'Nespamujte prosím. Váš účet byl na chvilku zablokován.'
     }
+	'he' : {
```

Comment:
Missing a comma before this block and a spurious one after, which is failing CI.